pulsar-client/src/main/java/org/apache/pulsar/client/impl/schema/AutoConsumeSchema.java

```diff
@@ -114,6 +117,21 @@ public SchemaInfo getSchemaInfo() {
         return schema.getSchemaInfo();
     }
 
+    @Override
+    public CompletableFuture<SchemaInfo> getSchemaInfo(byte[] schemaVersion) {
+        ensureSchemaInitialized();
+        if (schema.supportSchemaVersioning()) {
+            if (schemaVersion == null) {
```

Comment:
isn't is better to consistently return an error if schemaVersion is null ?

Reply:
good idea